velox/docs/configs.rst

```diff
@@ -203,6 +203,10 @@ Spilling
      - boolean
      - true
      - When `spill_enabled` is true, determines whether RowNumber operator can spill to disk under memory pressure.
+   * - writer_spill_enabled
+     - boolean
+     - true
+     - When `writer_spill_enabled` is true, determines whether TableWriter operator can spill to disk under memory pressure.
```

Comment:
"TableWriter operator can spill to disk"

Does spill to disk here refer to flushing buffered data to remote storage? If so, it would be nice to clarify as spilling usually means something else, i.e. writing data to temp storage for later processing.